saleor/plugins/base_plugin.py

```diff
@@ -711,6 +711,12 @@ def __str__(self):
     # metadata is changed.
     order_metadata_updated: Callable[["Order", Any], Any]
 
+    # Trigger when order is imported.
```

Comment:
Does this description is correct? It says `order is imported` it should be `orders` right?